ocs_ci/ocs/cephfs_workload.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf8 -*-
+
+import logging
+import os
+import time
+
+import yaml
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import exceptions
+from ocs_ci.ocs import ocp
+from ocs_ci.ocs.fio_artefacts import get_pvc_dict
+from ocs_ci.ocs.node import get_worker_nodes
+from ocs_ci.ocs.resources import job
+from ocs_ci.ocs.resources import topology
+from ocs_ci.ocs.resources.objectconfigfile import ObjectConfFile, link_spec_volume
+from ocs_ci.utility.utils import exec_cmd
+
+
+logger = logging.getLogger(__name__)
+
+
+class LogReaderWriterParallel(object):
+
+    """
+    This procedure in the test was originally created in the file
+    'tests/e2e/workloads/test_data_consistency.py'. I just rearranged it in a class.
+
+    Write and read logfile stored on cephfs volume, from all worker nodes of a
+    cluster via k8s Deployment, while fetching content of the stored data via
+    oc rsync to check the data locally.
+
+    """
+
+    def __init__(self, project, tmp_path):
+        self.project = project
+        self.tmp_path = tmp_path
+
+        self.pvc_dict = get_pvc_dict()
+        # we need to mount the volume on every worker node, so RWX/cephfs
+        self.pvc_dict["metadata"]["name"] = "logwriter-cephfs-many"
+        self.pvc_dict["spec"]["accessModes"] = [constants.ACCESS_MODE_RWX]
+        self.pvc_dict["spec"]["storageClassName"] = constants.CEPHFILESYSTEM_SC
+        self.pvc_dict["spec"]["resources"]["requests"]["storage"] = "2Gi"
+
+        self.deploy_dict = {}
+        self.workload_file = None
+        self.ocp_pod = None
+
+    def log_reader_writer_parallel(self):
```

Comment:
the implementation here is similar to test_data_consistency.py.
Please move it to a common place in order to avoid code duplication.
@mbukatov please review

Reply:
We can remove the file from the current pr, and include it in the next pr if it needs further configuration.